src/streaming/controllers/XlinkController.js

```diff
@@ -97,8 +97,13 @@ function XlinkController(config) {
         });
 
         manifest = mpd;
-        elements = getElementsToResolve(manifest.Period_asArray, manifest, DashConstants.PERIOD, RESOLVE_TYPE_ONLOAD);
-        resolve(elements, DashConstants.PERIOD, RESOLVE_TYPE_ONLOAD);
+
+        if (manifest.Period_asArray) {
```

Comment:
The specification correct semantics for Patching + XLink would be to apply the patch and then perform XLINK resolution, but given the load pathway changes that would take and the fact that XLink in live is undefined right now I felt it was better to bypass the XLink logic for now. Thoughts?

Reply:
I agree here, we would need to restructure the load workflow for this.